src/utils.ts

```diff
@@ -66,12 +66,14 @@ export async function checkModule(
     const [error] = await to(outputFixes(report))
 
     if (error) {
-      return Promise.reject(error)
+      throw error
```

Comment:
This is effectively the same thing as rejecting a promise within an `async` function.